sopel/bot.py

```diff
@@ -351,14 +394,20 @@ def say(self, text, recipient, max_messages=1):
     def notice(self, text, dest):
         """Send an IRC NOTICE to a user or a channel.
 
+        :param str text: the text to send in the NOTICE
+        :param str dest: the destination of the NOTICE
+
         Within the context of a triggered callable, ``dest`` will default to
         the channel (or nickname, if a private message), in which the trigger
         happened.
         """
         self.write(('NOTICE', dest), text)
 
     def action(self, text, dest):
-        """Send ``text`` as a CTCP ACTION PRIVMSG to ``dest``.
+        """Send a CTCP ACTION PRIVMSG to a user or a channel.
```

Comment:
```suggestion
        """Send a CTCP ACTION PRIVMSG to a user or channel.
```

Nitpicking, only because I already have other comments to make in this review, and I noticed inconsistencies.